src/5e-SRD-Features.json

```diff
@@ -97,6 +189,24 @@
     "desc": [
       "Starting when you choose this path at 3rd level, you can go into a frenzy when you rage. If you do so, for the duration of your rage you can make a single melee weapon attack as a bonus action on each of your turns after this one. When your rage ends, you suffer one level of exhaustion (as described in appendix A)."
     ],
+    "feature_specific": {
+      "prerequisites": [
+        {
+          "type": "boolean",
+          "from": "raging"
+        }
+      ],
+      "bonus_action": {
+        "action": "attack",
+        "amount": 1
+      },
+      "downsides": [
+        {
+          "type": "exhaustion",
```

Comment:
What about something more like the following?

```json
{
  "type": "condition",
  "condition": "exhaustion",
  "level": 1
}
```

Reply:
maybe an APIreference?

Reply:
Yeah, an APIReference to the condition would be ideal, actually!